docs/docs/gatsby-starters.md

```diff
@@ -962,3 +962,26 @@ gatsby new my-blog https://github.com/gatsbyjs/gatsby-starter-blog#v2
 
   - Built upon gatsby-starter-default
   - Perfect if you want to use Typescript and SASS but don't want any extra code
+  
+- [gatsby-starter-mate](https://github.com/EmaSuriano/gatsby-starter-mate)
+  [(demo)](https://gatsby-starter-mate.netlify.com/)
+
+  Features:
+
+  - Gatsby v2
+  - Rebass (Styled-components system)
+  - Dynamic content from Contentful CMS
+  - Offline
+  - Lighthouse score of 93
+  - Offline support
```

Comment:
You have both `Offline` and `Offline support` - should one be removed?

Reply:
Ups sorry! Fixed.